### ISSUE TITLE
Add autoscaling flags

### ISSUE THREAD

**Issue by matthew1534** [cppforlife/knctl#61]
Would it be possible to add the ability to change the basic autoscaling parameters i.e. `containerConcurrency`, `minScale` & `maxScale` when using the `knctl deploy` command?

Currently we're deploying an app (which is very easy thanks to knctl!) but changing the autoscaling parameters is a pain. At the moment we do `kubectl apply` for the below code however this doesn't always seem to get applied (has worked twice out of 8 separate occasions) / can't be re-deployed new using knctl - says it is deploying with no other messages and never actual deploys, think the issue might be to do with `kubectl apply` command not switching the flags `latest` and `previous`.

```apiVersion: serving.knative.dev/v1alpha1
kind: Service
metadata:
  name: test
  namespace: development
spec:
  runLatest:
    configuration:
      revisionTemplate:
        metadata:
          annotations:
            # Maximum number of pods
            autoscaling.knative.dev/maxScale: "90"
            # Minimum number of pods
            autoscaling.knative.dev/minScale: "0"
        spec:
          containerConcurrency: 5
          container:
            image: gcr.io/project/app-name
            env:
            - name: NODE_ENV
              value: development
````

I'm relatively new to Kubernetes & Knative but it seems like everything else is very easy and this seems a natural progression but it is more than possible that I've missed something RE changing the autoscaling parameters so please do close the issue if you think this is irrelevant!

Really appreciate the fab work that has been done!!

**Comment by cppforlife** [comment]
yeah, this makes sense to add as a feature. will probably get to it in next few days. feel free to send a PR if you are adventurous (it would go in here i believe: https://github.com/cppforlife/knctl/blob/master/pkg/knctl/cmd/service/service_spec.go#L129)

**Comment by cppforlife** [comment]
added in https://github.com/cppforlife/knctl/commit/62174e1741cad07473e938b8c318d380f1a3507e. going thru ci.

**Comment by cppforlife** [comment]
available in v0.3.0
